### PR TITLE
better isort defaults, rerun, reblacked

### PR DESCRIPTION
**Pull request by nicpottier**
Trailing comma in black is the only difference now, could be worth doing this until a new version of black comes out:
  https://github.com/ambv/black/issues/250

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -3,3 +3,13 @@ max-line-length=120
...
 exclude=./env/*,./config/*,./fab*,.git/*,static/bower/*,sitestatic/bower/,*/gen/*
 ignore=E501,F405,T003,E203,W503
+
+[isort]
+multi_line_output = 3
+including_trailing_comma = True
+force_grid_wrap = 0
+line_length = 119
+combine_as_imports = True
+sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOTHIRDPARTY,FIRSTPARTY,LOCALFOLDER
```
Comment (rowanseymour) [inline setup.cfg]: not sure I'd bother splitting third party into THIRDPARTY,DJANGO,DJANGOTHIRDPARTY but no biggie

Comment (nicpottier) [reply setup.cfg]: Not sure it is actually being triggered anywhere

**Comment by nicpottier** [comment]
Found that with `trailing_comma` option they now agree. I think that ticket only applies for super long top level package names which we don't run into, so I think we can merge this as is.

File: setup.cfg (hunk 1 above)
Comment (rowanseymour) [reply setup.cfg]: all the files I'm seeing now have django.* imports in their own section

Comment (nicpottier) [reply setup.cfg]: Oh thought you were talking third party django vs django. I for one like having Django being in their own thing.

This is why we shouldn't have options!

File: setup.cfg [hunk 2]
```diff
@@ -3,3 +3,14 @@ max-line-length=120
...
+[isort]
+multi_line_output = 3
+including_trailing_comma = True
+force_grid_wrap = 0
+line_length = 119
+include_trailing_comma = True
+combine_as_imports = True
+sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOTHIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_django = django
+known_djangothirdparty = celery, leaflet
```
Comment (norkans7) [inline setup.cfg]: I think smartmin, DRF, django-countries should also belong in the django third party

**Pull request merged**